### PR TITLE
Feature/ics

### PR DESCRIPTION
**Pull request by AshenSilverwolf**
When a user creates a reservation, they will receive an email with an attached .ics file containing the date/time information of said reservation.

### CODE REVIEW

File: src/routes/reservation.py [hunk 1]
```diff
@@ -293,6 +298,43 @@ def create_reservation(**kwargs):
...
+                post_data["email"],
+                "CHDR Item Reservation Confirmation",
+                email_body,
+                ics_path,
+            )
+            # Emailer.send_email("chdr email here", "CHDR Item Reservation", body, ics_path)
+        except SMTPException as e:
+            current_app.logger.error(e.message)
+
+        os.remove(ics_path)
```
Comment (ctcuff) [inline src/routes/reservation.py:332]: Might wanna wrap this in a `try` block just in case it throws an error for some reason.

**Review by ctcuff — COMMENTED** [review]
Looks and works great but we should have the ics show the name of the item in the description. Just saying `item reservation` is a little vague.

File: src/routes/reservation.py (hunk 1 above)
Comment (AshenSilverwolf) [reply src/routes/reservation.py]: 051aba3

**Review by ctcuff — APPROVED** [review]
Nice! 👍

**Pull request merged**